### PR TITLE
https://github.com/JetBrains/kotlin-native/pull/3538 followup.

### PR DESCRIPTION
**Pull request by olonho**

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -356,7 +356,8 @@ task distDef(type: Copy) {
     destinationDir distDir
 
     from(project("platformLibs").file("src/platform")) {
-        into('klib/platformDef')
+        into('konan/platformDef')
+        exclude '**/*.sh'
```
Comment (SvyatoslavScherbina) [inline build.gradle]: Why not `include '**/*.def'` instead?

File: build.gradle [hunk 2]
```diff
@@ -461,6 +462,7 @@ task crossDist {
 }
 
 task bundle(type: (isWindows()) ? Zip : Tar) {
+    dependsOn("distDef")
```
Comment (SvyatoslavScherbina) [inline build.gradle]: This is likely required for both `bundle` and `bundleRestricted`.

File: utilities/src/main/kotlin/org/jetbrains/kotlin/cli/utilities/GeneratePlatformLibraries.kt [hunk 3]
```diff
@@ -83,6 +83,7 @@ private fun generatePlatformLibraries(target: String, inputDirectory: File, outp
         if (!saveTemps)  {
             File("$outputDirectory/build-${def.name}").deleteRecursively()
         }
+        File("$outputDirectory/clangModulesCache").deleteRecursively()
```
Comment (SvyatoslavScherbina) [inline utilities/src/main/kotlin/org/jetbrains/kotlin/cli/utilities/GeneratePlatformLibraries.kt]: The idea behind module cache here was to reuse it for all platform libraries.

Comment (olonho) [reply utilities/src/main/kotlin/org/jetbrains/kotlin/cli/utilities/GeneratePlatformLibraries.kt]: Yeah, moved removal below, PTAL.

**Pull request merged**